{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -48,6 +48,20 @@
     },
 }
 
+
+REDIS_URL = env("REDIS_TLS_URL", default=env("REDIS_URL"))
+REDIS_SSL = env.bool("REDIS_SSL", default=False)
```

Comment:
This should probably be in `base.py` and can be derived from the `REDIS_URL`, I think

```suggestion
REDIS_SSL = REDIS_URL.startswith("rediss://")
```